src/nvim/ui.c

```diff
@@ -439,7 +439,7 @@ void ui_line(ScreenGrid *grid, int row, int startcol, int endcol, int clearcol,
     ui_call_grid_cursor_goto(grid->handle, row,
                              MIN(clearcol, (int)grid->cols - 1));
     ui_call_flush();
-    uint64_t wd = (uint64_t)labs(p_wd);
+    uint64_t wd = (uint64_t)labs((long)p_wd);
```

Comment:
`long long` is guaranteed to be at least 64 bits, so you can just use `llabs` here and remove the cast
```suggestion
    uint64_t wd = (uint64_t)llabs(p_wd);
```